sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/datasketchesAggregates.scala

```diff
@@ -234,6 +237,9 @@ case class HllUnionAgg(
     right.eval().asInstanceOf[Boolean]
   }
 
+  // The default target type (register size) to use.
+  private val targetType = TgtHllType.HLL_8
```

Comment:
this is duplicated with L71 above, can we dedup to one place? Same for the change in `datasketchesExpressions.scala`

Reply:
Let's do it in a follow up PR. I prefer that we merge this change ASAP and then figure out how to refactor code properly.